### PR TITLE
[CS] Add `Welcome`

### PR DESCRIPTION
**Pull request by Ondra-D**
Translated the welcome osu! wiki page to czech

### CODE REVIEW

**Comment by DoryanR** [comment]
I think translating this page is not a good idea, as it plans to be deleted on #5506

File: wiki/Welcome/cs.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Welcome
+
+Výtej v osu!, free-to-win rhythm hře vyvinutá peppym se čtyřmi herními módy: osu! mode, simulátor klikání na kolečka; osu!taiko, emulátor bubnování; osu!catch, chytač ovocného salátu; a osu!mania, syntetizátor rozbíjení kláves.
+
+
```
Comment (TPGPL) [inline wiki/Welcome/cs.md:5]:
```suggestion
```

File: wiki/Welcome/cs.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## Common points of interest
```
Comment (TPGPL) [inline wiki/Welcome/cs.md:6]: needs to be translated

File: wiki/Welcome/cs.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+- [Instalace](/wiki/Installation)
+- [Registrace](/wiki/Registration)
+- [Pravidla](/wiki/Rules)
+- [FAQ](/wiki/FAQ)
+- [Herní módy](/wiki/Game_mode)
+  - ![osu! mode icon](/wiki/shared/mode/osu.png) [osu!](/wiki/Game_mode/osu!)
+  - ![osu!taiko icon](/wiki/shared/mode/taiko.png) [osu!taiko](/wiki/Game_mode/osu!taiko)
+  - ![osu!catch icon](/wiki/shared/mode/catch.png) [osu!catch](/wiki/Game_mode/osu!catch)
+  - ![osu!mania icon](/wiki/shared/mode/mania.png) [osu!mania](/wiki/Game_mode/osu!mania)
```
Comment (TPGPL) [inline wiki/Welcome/cs.md:16]: image alt-texts need to be translated

File: wiki/Welcome/cs.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- [Multi](/wiki/Multi)
+- [Kritéria hodnocení](/wiki/Ranking_Criteria)
+  - ![osu! icon](/wiki/shared/mode/osu.png) [osu!](/wiki/Ranking_Criteria/osu!)
+  - ![osu!taiko icon](/wiki/shared/mode/taiko.png) [osu!taiko](/wiki/Ranking_Criteria/osu!taiko)
+  - ![osu!catch icon](/wiki/shared/mode/catch.png) [osu!catch](/wiki/Ranking_Criteria/osu!catch)
+  - ![osu!mania icon](/wiki/shared/mode/mania.png) [osu!mania](/wiki/Ranking_Criteria/osu!mania)
```
Comment (TPGPL) [inline wiki/Welcome/cs.md:22]: image alt-texts need to be translated

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Welcome/cs.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  - ![osu! ikona](/wiki/shared/mode/osu.png) [osu!](/wiki/Ranking_Criteria/osu!)
+  - ![osu!taiko ikona](/wiki/shared/mode/taiko.png) [osu!taiko](/wiki/Ranking_Criteria/osu!taiko)
+  - ![osu!catch ikona](/wiki/shared/mode/catch.png) [osu!catch](/wiki/Ranking_Criteria/osu!catch)
+  - ![osu!mania ikona](/wiki/shared/mode/mania.png) [osu!mania](/wiki/Ranking_Criteria/osu!mania)
+
+## Beatmapping
+
+*Main page: [Beatmapping](/wiki/Beatmapping)*
+
+Beatmapping je proces, kdy tvůrce vytváří beatmapu. Tento proces zahrnuje: výběr hudby, načasování a mapování beatmapy a testování beatmapy. Může také zahrnovat: přidání videa, přidání [storyboardu](/wiki/Storyboarding), a / nebo přidání [vlastního vzledu](/wiki/Skinning). To vše může tvůrce provést sám nebo s dalšími uživateli, někdy označovanými jako * storyboarders *, * skinners * a / nebo * guest creators *, podle toho, co dělají.
```
Comment (TPGPL) [inline wiki/Welcome/cs.md:27]:
```suggestion
Beatmapping je proces, kdy tvůrce vytváří beatmapu. Tento proces zahrnuje: výběr hudby, načasování a mapování beatmapy a testování beatmapy. Může také zahrnovat: přidání videa, přidání [storyboardu](/wiki/Storyboarding), a / nebo přidání [vlastního vzledu](/wiki/Skinning). To vše může tvůrce provést sám nebo s dalšími uživateli, někdy označovanými jako *storyboarders*, *skinners* a / nebo *guest creators*, podle toho, co dělají.
```

File: wiki/Welcome/cs.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+## Modding
+
+*Main page: [Modding](/wiki/Modding)*
```
Comment (TPGPL) [inline wiki/Welcome/cs.md:31]: `Main page` needs to be translated

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**